glue/config.py

```diff
@@ -35,6 +34,7 @@ class Registry(object):
 
     def __init__(self):
         self._members = []
+        self._plugins = []
```

Comment:
Actually, should this then just be a public attribute and we get rid of ``lazy_add``, so that users can append but also remove from ``lazy_members``?